README.md

```diff
@@ -25,6 +25,21 @@ installed on your system, as it's used to manage some large files stored in
 GitHub.
 
 
+## Running the Tests in GitHub:
```

Comment:
I think we also need guidance for how someone would add their regression test suite to the GitHub CI/CD. (And a great test would then be to get a couple of service providers and data providers to try to follow those instructions for existing test suites)

Reply:
Also, perhaps discussion whether/how a regression test suite could live in an external repository would be valuable. (It might not be possible, in which case, that's a simple thing to say)